agent/engine/engine_unix_integ_test.go

```diff
@@ -1227,7 +1227,7 @@ func TestHostResourceManagerResourceUtilization(t *testing.T) {
 		testTask := createTestTask(taskArn)
 
 		// create container
-		A := createTestContainerWithImageAndName(baseImageForOS, "A")
+		A := createTestContainerWithImageAndName(baseImageForOS, fmt.Sprintf("A-%d", i))
```

Comment:
This is an unrelated change in another test case related to task resource accounting. Can make debugging easier in future if each task and container are named uniquely.